core/src/sidebar/sidebar.module.css

```diff
@@ -0,0 +1,10 @@
+.container {
+	width: 232px;
```

Comment:
Can I customizable this property?

Reply:
yes because it is used for the div that wrap around the main content in sidebar.stories